### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by naiteeks**
minor formatting changes

### CODE REVIEW

File: MediaGraph/topologies/evr-hubMessage-files/readme.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
...
+This topology enables you to record video clips to the local file system of the edge device whenever an external sensor sends a message to the Media Graph. You can read more about this scenario in the documentation page. The diagram below shows a door sensor but it could be any sensor or app sending the message.  
 
-Note: the topology creates new MP4 files each time the event trigger is received. Over time, this can fill up the local filesystem. You should monitor the contents of the output directory and prune older files as necessary.
+Note: This topology is very similar to the topology where you recorded video clips only when desired objects were detected. Similar to the ObjectCounter module in that tutorial, you will need an external module for this example, update the desired properties to look for specificevents and declare routes to send those events to the IoT Hub Source node.
```
Comment (anilmur) [inline MediaGraph/topologies/evr-hubMessage-files/readme.md:5]: This topology is very similar to the [topology](..\..\that folder\readme.md) where you recorded ...

File: MediaGraph/topologies/httpExtension/readme.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
-# Analyzing live video using HTTP Extension to send images to an external inference engine 
+# Analyzing live video using HTTP Extension to send images to an external inference engine.   
 
-This topology enables you to run video analytics on a live feed from an RTSP-capable camera. A subset of the video frames from the camera (as controlled by the Frame Rate Filter) are converted to images, and sent to an external AI inference engine. The results are then published to the IoT Hub.
+This topology enables you to run video analytics on a live feed from a RTSP-capable camera. A subset of the video frames from the camera (as controlled by the Frame Rate Filter) are converted to images, and sent to an external AI inference engine. The results are then published to the IoT Hub.
```
Comment (anilmur) [inline MediaGraph/topologies/httpExtension/readme.md:3]: See https://dictionary.cambridge.org/grammar/british-grammar/a-an-and-the

Just like with "an MP3 player or an MP4 file", we have "an RTSP source"

File: MediaGraph/topologies/evr-hubMessage-files/readme.md (hunk 1 above)
Comment (naiteeks) [reply MediaGraph/topologies/evr-hubMessage-files/readme.md]: fixed with commit: 7e94cb2

File: MediaGraph/topologies/httpExtension/readme.md (hunk 2 above)
Comment (naiteeks) [reply MediaGraph/topologies/httpExtension/readme.md]: fixed with commit: 7e94cb2

File: MediaGraph/topologies/evr-hubMessage-files/readme.md [hunk 3]
```diff
@@ -1,8 +1,9 @@
...
+This topology enables you to record video clips to the local file system of the edge device whenever an external sensor sends a message to the Media Graph. You can read more about this scenario in the documentation page. The diagram below shows a door sensor but it could be any sensor or app sending the message.  
 
-Note: the topology creates new MP4 files each time the event trigger is received. Over time, this can fill up the local filesystem. You should monitor the contents of the output directory and prune older files as necessary.
+Note: This topology is very similar to the [topology](../evr-hubMessage-assets/topology.json) where you recorded video clips only when desired objects were detected. Similar to the ObjectCounter module in that tutorial, you will need an external module for this example, update the desired properties to look for specificevents and declare routes to send those events to the IoT Hub Source node.  
+Also, note that over time, this can fill up the local filesystem. You should monitor the contents of the output directory and prune older files as necessary.
```
Comment (gadamilan) [inline MediaGraph/topologies/evr-hubMessage-files/readme.md:6]: There are some errors in the description. 

This page is not an example. It is a readme for a topology. Hence, you need to describe this with that context.

Which desired properties are you referring to?

Be clear that you are referring to routes in IoT Edge deployment manifest (remember this is a topology page).

The line about filling up local file system is not required (this is not a tutorial or sample page)

**Pull request merged**